ui/pages/home/home.container.js

```diff
@@ -138,7 +138,8 @@ const mapStateToProps = (state) => {
     selectedAddress,
     firstPermissionsRequestId,
     totalUnapprovedCount,
-    hasApprovalFlows: getApprovalFlows(state).length > 0,
+    hasApprovalFlows:
+      Array.isArray(getApprovalFlows) && getApprovalFlows(state).length > 0,
```

Comment:
Can i ask under which condition `getApprovalFlows` is not an array?

Reply:
The mmi build failed because `.length` was undefined, so I had to add this check. This is unrelated at mmi, and it was introduced some weeks ago. So it was good to add that check. Let me know if there's something else I can do

Reply:
@jpuri I'd actually like to get your eyes on this or someone from confirmations team (if I should be tagging someone else let me know)

Reply:
Thanks a lot @brad-decker , yes we will ensure we review it earliest.